vector-im/element-android
# [Voice Broadcast] Update buffering display and improve playback

## Type of change


## Content

- Move the buffering view to the timeline header
- Display the `pause` button when the voice broadcast is buffering to let the user stop the buffering
- Some UI adjustments (padding, colors)
- Fixed playback issues

## Motivation and context

Design changes:

<img width="602" alt="image" src="https://user-images.githubusercontent.com/11990514/204407753-6bbc29cd-61c9-4f80-b70f-f7c723526038.png">

## Screenshots / GIFs

## Tests

1. Start recording a VB from device A
2. Start listening to the VB from device B
3. Observe that the buffering is displayed as expected while waiting for a new chunk
4. Verify that the pause button work as expected during the buffering

## Tested devices

- OS version(s):

## Checklist
